.rubocop.yml

```diff
@@ -14,136 +14,7 @@ AllCops:
-  Enabled: true
-Style/MinMaxComparison: # new in 1.42
-  Enabled: true
-Style/NestedFileDirname: # new in 1.26
-  Enabled: true
-Style/ObjectThen: # new in 1.28
-  Enabled: true
-Style/OperatorMethodCall: # new in 1.37
-  Enabled: true
-Style/RedundantConstantBase: # new in 1.40
-  Enabled: true
-Style/RedundantDoubleSplatHashBraces: # new in 1.41
-  Enabled: true
-Style/RedundantEach: # new in 1.38
-  Enabled: true
-Style/RedundantInitialize: # new in 1.27
-  Enabled: true
-Style/RedundantStringEscape: # new in 1.37, 'pending' by default so enabled to make sure it's applied
-  Enabled: true
-Style/YodaExpression: # new in 1.42
-  Enabled: true
-Minitest/AssertKindOf: # new in 0.10
-  Enabled: true
-Minitest/AssertOutput: # new in 0.10
-  Enabled: true
-Minitest/AssertPathExists: # new in 0.10
-  Enabled: true
-Minitest/AssertPredicate: # new in 0.18
-  Enabled: true
-Minitest/AssertRaisesCompoundBody: # new in 0.21
-  Enabled: true
-Minitest/AssertRaisesWithRegexpArgument: # new in 0.22
-  Enabled: true
-Minitest/AssertSilent: # new in 0.10
-  Enabled: true
-Minitest/AssertWithExpectedArgument: # new in 0.11
-  Enabled: true
-Minitest/AssertionInLifecycleHook: # new in 0.10
-  Enabled: true
-Minitest/DuplicateTestRun: # new in 0.19
-  Enabled: true
-Minitest/EmptyLineBeforeAssertionMethods: # new in 0.23
-  Enabled: true
-Minitest/LiteralAsActualArgument: # new in 0.10
-  Enabled: true
-Minitest/MultipleAssertions: # new in 0.10
-  Enabled: true
-Minitest/RefuteInDelta: # new in 0.10
-  Enabled: true
-Minitest/RefuteKindOf: # new in 0.10
-  Enabled: true
-Minitest/RefutePathExists: # new in 0.10
-  Enabled: true
-Minitest/SkipEnsure: # new in 0.20
-  Enabled: true
-Minitest/SkipWithoutReason: # new in 0.24
-  Enabled: true
-Minitest/UnreachableAssertion: # new in 0.14
-  Enabled: true
-Minitest/UnspecifiedException: # new in 0.10
-  Enabled: true
-Performance/AncestorsInclude: # new in 1.7
-  Enabled: true
-Performance/BigDecimalWithNumericArgument: # new in 1.7
-  Enabled: true
-Performance/BlockGivenWithExplicitBlock: # new in 1.9
-  Enabled: true
-Performance/CollectionLiteralInLoop: # new in 1.8
-  Enabled: true
-Performance/ConcurrentMonotonicTime: # new in 1.12
-  Enabled: true
-Performance/ConstantRegexp: # new in 1.9
-  Enabled: true
-Performance/RedundantEqualityComparisonBlock: # new in 1.10
-  Enabled: true
-Performance/RedundantSortBlock: # new in 1.7
-  Enabled: true
-Performance/RedundantSplitRegexpArgument: # new in 1.10
-  Enabled: true
-Performance/RedundantStringChars: # new in 1.7
-  Enabled: true
-Performance/ReverseFirst: # new in 1.7
-  Enabled: true
-Performance/SortReverse: # new in 1.7
-  Enabled: true
-Performance/Squeeze: # new in 1.7
-  Enabled: true
-Performance/StringIdentifierArgument: # new in 1.13
-  Enabled: true
-Performance/StringInclude: # new in 1.7
-  Enabled: true
-Performance/Sum: # new in 1.8, pending so left enabled explicitly until 2.0
-  Enabled: true
-
-# Old cops
+  NewCops: enable
```

Comment:
We're currently thinking that an exact version lock of the RuboCop version will work best with enabling all new cops each time and not having to explicitly list all of the new ones.